### DISCUSSION TITLE
What does 'LM' suffix mean?

### DISCUSSION THREAD

**Discussion by Wyswyss** [wenet-e2e/wespeaker#329] [Q&A] [answered]
https://github.com/wenet-e2e/wespeaker/blob/master/docs/pretrained.md
There're lots of models having "LM" suffixes. What does that mean? Thanks!

**Comment by JiJiJiang** [comment, marked as answer]
The corresponding model after Large-Margin finetuning.
We will mark this in the doc. Thanks for reminding.
